tests/conftest.py

```diff
@@ -8020,19 +8025,38 @@ def create_hypershift_clusters():
     """
 
     def factory(
-        cluster_names, ocp_version, odf_version, setup_storage_client, nodepool_replicas
+        num_of_clusters=1,
```

Comment:
this is redundunt, we already have cluster_names - len() makes the job

Reply:
Fixed

Reply:
thanks